sip-test-kit/src/main/java/de/ikor/sip/foundation/testkit/workflow/whenphase/routeinvoker/headers/FileExchangeHeaders.java

```diff
@@ -0,0 +1,26 @@
+package de.ikor.sip.foundation.testkit.workflow.whenphase.routeinvoker.headers;
+
+/** Enum with Camel File, FTP, FTPS, SFTP exchange headers which we can set programmatically */
+public enum FileExchangeHeaders {
+  CAMEL_FILE_NAME("CamelFileName"),
+  CAMEL_FILE_LENGTH("CamelFileLength"),
+  CAMEL_FILE_NAME_CONSUMED("CamelFileNameConsumed"),
+  CAMEL_FILE_NAME_ONLY("CamelFileNameOnly"),
+  CAMEL_FILE_LAST_MODIFIED("CamelFileLastModified"),
+  CAMEL_MESSAGE_TIMESTAMP("CamelMessageTimestamp");
```

Comment:
Can we make the connection to camel here? Reference Exchange.FILE_LENGTH and similar costants?

Reply:
Yes, there are these headers in camel Exchange class. Refactored according to your comment.